File: c++/20140102/Day03MyDate_my副本/Day03MyDate_my/MyDate.cpp
#include "MyDate.h"
#include <iostream>
using namespace std;

MyDate::MyDate(int year,int m,int d){
    this->year = year;
    month = m;
    day = d;
}

void MyDate::show(){
    cout<<year<<"-"<<month<<"-"<<day<<endl;
}

File: c++/20140102/text/text/Mydate.m
//
//  Mydate.m
//  text
//
//  Created by HanyFeng on 14-1-4.
//  Copyright (c) 2014年 Hany. All rights reserved.
//

#import "Mydate.h"

@implementation Mydate

-(id)initWithYear:(int)year andMon:(int)mon andDay:(int)day{
    if (self = [super init]) {
        self.year = year;
        self.mon = mon;
        self.day = day;
    }
    return self;
}

-(void)show{
    NSLog(@"%d-%d-%d",self.year,self.mon,self.day);
}

-(void)addDay{
    self.day++;
}
@end
